guides/image-sources/provisioning-image-finish.plantuml

```diff
@@ -0,0 +1,47 @@
+@startuml
+
+!include foreman.pstyle
+
+title Image-based provisioning with Finish script configuration
+
+actor       User
+participant "Provisioned\ninstance" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant "Infrastructure\ncloud" as Cloud
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Foreman : Has an image\nwith SSH credentials\ndefined in compute resource
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host\nwith a cloud resource and the image
+group Template [Finish]
```

Comment:
This step is not happening. We render the template directly from the Foreman for the host via SSH, after the host has booted.